libjlm/src/tooling/CommandGraphGenerator.cpp

```diff
@@ -114,12 +114,31 @@ JlcCommandGraphGenerator::GenerateCommandGraph(const JlcCommandLineOptions & com
 
     if (compilation.RequiresOptimization())
     {
+      std::vector<JlmOptCommand::Optimization> optimizations;
+      if (!commandLineOptions.JlmOptOptimizations_.empty()) {
+        static std::unordered_map<std::string, JlmOptCommand::Optimization>map(
+        {
+	  {"AASteensgaardBasic", JlmOptCommand::Optimization::AASteensgaardBasic},
+	  {"cne", JlmOptCommand::Optimization::CommonNodeElimination},
+	  {"dne", JlmOptCommand::Optimization::DeadNodeElimination},
+	  {"iln", JlmOptCommand::Optimization::FunctionInlining},
+	  {"InvariantValueRedirection", JlmOptCommand::Optimization::InvariantValueRedirection},
+	  {"psh", JlmOptCommand::Optimization::NodePushOut},
+	  {"pll", JlmOptCommand::Optimization::NodePullIn},
+	  {"red", JlmOptCommand::Optimization::NodeReduction},
+	  {"ivt", JlmOptCommand::Optimization::ThetaGammaInversion},
+	  {"url", JlmOptCommand::Optimization::LoopUnrolling},
+        });
+        for (const auto & jlmOpt : commandLineOptions.JlmOptOptimizations_)
+	{
+          JLM_ASSERT(map.find(jlmOpt) != map.end());
+          optimizations.push_back(map[jlmOpt]);
+	}
```

Comment:
Could we fix the identation please?

Reply:
Indentation and test should be fixed

Reply:
The parentheses of the for loop are still not properly indented.